core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -1564,7 +1564,7 @@ object LogManager {
       Option(newTopicsImage.getPartition(topicId, partitionId)) match {
         case Some(partition) =>
           if (!partition.replicas.contains(brokerId)) {
-            info(s"Found stray log dir $log: the current replica assignment ${partition.replicas} " +
+            info(s"Found stray log dir $log: the current replica assignment ${partition.replicas.mkString("[", ", ", "]")} " +
```

Comment:
are we assuming 3 replicas here?

Reply:
Does it make sense to be
[${{partition.replicas.mkString(",")}]

Reply:
Why? the `"["` is the starting string, `", "` is the separator and `"]"` is the ending string. `Array(1,2).mkString("[", ", ", "]")` returns `"[1, 2]"`.

Reply:
Ok. I guess our suggestions are the same 🤷‍♀️ . It was a bit less readable to me which is why I was confused.

Reply:
^ Agreed, adding the brackets outside makes it more readable.